app/assets/javascripts/school_administrator_dashboard/students_table.jsx

```diff
@@ -91,10 +96,13 @@ export default React.createClass({
           <tbody>
             {this.orderedRows().map(student => {
               return (
-                <tr>
-                  <td>{student.last_name}</td>
-                  <td>{student.first_name}</td>
-                  <td>{student.absences}</td>
+                <tr key={student.id}>
+                  <td width="66.66%">
```

Comment:
This is fine, just as a side note we use `flex` for this in other places in the app.